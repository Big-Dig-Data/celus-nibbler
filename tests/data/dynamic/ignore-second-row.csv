IIII,,
2020-01,2020-02,2020-03
1,2,3
4,5,6
