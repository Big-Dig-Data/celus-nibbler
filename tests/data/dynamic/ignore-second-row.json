{
  "kind": "non_counter.generic",
  "version": 1,
  "parser_name": "ignore_second_row",
  "data_format": {
    "name": "ignore_second_row"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [],
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "IIII",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                      "row": 1,
                      "col": 0
                    },
                    "direction": "right"
                },
                "role": "date"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 2,
            "col": 0
          },
          "direction": "right"
        },
        "data_extract_params": {
            "max_idx": 0
        }
      },
      "titles": null,
      "metrics": {
        "source": {
          "value": "Met"
        }
      },
      "title_ids": [],
      "dimensions": [],
      "kind": "non_counter.generic"
    }
  ]
}
