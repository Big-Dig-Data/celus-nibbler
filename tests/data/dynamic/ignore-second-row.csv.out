"2020-01-01","2020-01-31","","","Met","","","1"
"2020-02-01","2020-02-29","","","Met","","","2"
"2020-03-01","2020-03-31","","","Met","","","3"
